content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+            port: 15443
+EOF
+{{< /text >}}
+
+As we did with `FRED` above, install a gateway in `BARNEY` that is dedicated
+to east-west traffic and expose user services.
+
+{{< text bash >}}
+$ CLUSTER=BARNEY NETWORK=EAST \
+    samples/multicluster/gen-eastwest-gateway.sh | \
+    kubectl apply --context=${CTX_BARNEY} -f -
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl --context=${CTX_BARNEY} apply -n istio-system -f \
+    samples/multicluster/expose-services.yaml
+{{< /text >}}
+
+### Enable Endpoint Discovery for `FRED` and `BARNEY`
+
+Install a remote secret in `BARNEY` that provides access to `FRED`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_FRED} \
+  --name=FRED | \
+  kubectl apply -f - --context=${CTX_BARNEY}
+{{< /text >}}
+
+Install a remote secret in `FRED` that provides access to `BARNEY`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_BARNEY} \
+  --name=BARNEY | \
+  kubectl apply -f - --context=${CTX_FRED}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Primary-Remote" category-value="primary-remote" >}}
+
+The following steps will install the Istio control plane on `FRED` (the
+{{< gloss >}}primary cluster{{< /gloss >}}) and configure `BARNEY` (the
+{{< gloss >}}remote cluster{{< /gloss >}}) to use the control plane in `FRED`. Both
+clusters reside on the `EAST` network.
+
+`FRED` will be configured to observe the API Servers in both clusters for
+endpoints. In this way, the control plane will be able to provide service
+discovery for workloads in both clusters. Service workloads across cluster
+boundaries communicate indirectly, via dedicated
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateways.
+
+{{< image width="75%"
+    link="primary-remote.svg"
+    caption="Primary and remote clusters on the same network"
+    >}}
+
+### Configure `FRED` as a primary with control plane exposed
+
+Install Istio on `FRED`:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_FRED} -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: FRED
+      network: EAST
+      meshNetworks:
+        ${NETWORK1}:
+          endpoints:
+          - fromRegistry: FRED
+          - fromRegistry: BARNEY
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+Install a gateway in the `FRED` that is dedicated to
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) traffic. By
+default, this gateway will be public on the Internet. Production systems may
+require additional access restrictions (e.g. via firewall rules) to prevent
+external attacks. Check with your cloud vendor to see what options are
+available.
+
+{{< text bash >}}
+$ CLUSTER=FRED NETWORK=WEST \
```

Comment:
We don't need the E-W gateway (there is no WEST anyway).

Reply:
We're using the term E-W here to describe the traffic direction, it's not referring to the network names.

It is a bit confusing since we named our networks EAST and WEST. I guess we could rename the gateway to something like `in-mesh` or `inter-cluster`, but `east-west` is a fairly common term for this. Perhaps we rename the networks in these examples?